### DISCUSSION TITLE
MongoDB Time Series

### DISCUSSION THREAD

**Discussion by jochman** [sqlalchemy/sqlalchemy#9646] [Usage Questions]
I couldn't find an answer for that - Is sqlalchemy supports mongodb timeseries feature?

if so, how do I create a table and query on time series?

**Comment by zzzeek** [comment]
SQLAlchemy does not have an included mongodb dialect.     a quick google for third party didnt find much either.  if you're using mongodb you should use the native python libraries for that database.
